frontend/discourse/app/instance-initializers/message-bus.js

```diff
@@ -31,6 +31,9 @@ function mbAjax(messageBus, opts) {
 
   if (_sendDeferredPageview) {
     opts.headers["Discourse-Deferred-Track-View"] = "true";
+    opts.headers["Discourse-Tracking-Session-Id"] = messageBus.clientId;
+    opts.headers["Discourse-Tracking-Url"] = window.location.href;
+    opts.headers["Discourse-Tracking-Referrer"] = document.referrer;
```

Comment:
```suggestion
    opts.headers["Discourse-Deferred-Track-View-Session-Id"] = messageBus.clientId;
    opts.headers["Discourse-Deferred-Track-View-Url"] = window.location.href;
    opts.headers["Discourse-Deferred-Track-View-Referrer"] = document.referrer;
```

I think we should avoid introducing another prefix given that we already have `Discourse-Deferred-Track-View-Topic-Id`.

Reply:
Maybe we just centralise everything in this PR into one prefix? 

`Discourse-Track-View: 1` and `Discourse-Track-View-*` for the other headers. To be honest, I'm not even sure why we need to differentiate between deferred track view and the normal ones but we could add `Discourse-Track-View-Deferred: 1` for the server to differentiate between stuff.